.env.sample

```diff
@@ -30,6 +31,7 @@ KINTO_USER=experimenter
 LOGGING_CONSOLE_LEVEL=INFO
 LOGGING_USE_JSON=False
 NORMANDY_API_HOST=
+NORMANDY_DEVTOOLS_HOST=https://normandy-devtools.services.mozilla.com/
```

Comment:
Will this be different for different deployments?  Or will it always point to the prod one?

Reply:
I don't think it should be different. stage and prod could point to the prod one. One could navigate to stage recipes with the prod devtools instance.

Reply:
If it's always the same then you can just move it right into settings.py and not make it an env var at all?